tools/dependencies/known-dependencies.txt

```diff
@@ -28,7 +28,7 @@ apacheds-i18n-2.0.0-M15.jar
 apacheds-kerberos-codec-2.0.0-M15.jar
 api-asn1-api-1.0.0-M20.jar
 api-util-1.0.0-M20.jar
-apiguardian-api-1.1.0.jar
+apiguardian-api-1.1.2.jar
```

Comment:
hi, Ilooked this PR and it's good overall, but I have a question, the jars we introduce are all at the test level, why is there a jar upgrade? Can you check where it was imported?

Reply:
It upgrades because of `junit-jupiter-engine` in Junit 5

Reply:
got it. It seems that other components also depend on this jar. Please follow this document to add the corresponding license changes.

Reply:
https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md